beta/azure-container-apps/ReadMe.md

```diff
@@ -248,15 +272,221 @@ The `scimsession` credentials will be saved as a secret variable in Container Ap
 
 ## Appendix
 
-### Troubleshooting
+### Resource recommendations
```

Comment:
Love that we're including this. I see the CLI commands below appear to allocate more resources to the _containers_, will that automatically scale up the resources of the container environment itself? Or would they need to separately provision more resources to the environment to be equal to or grater than the total resources allocated to each container?

Reply:
The nice thing about Container Apps, is that the update option applied through the GUI or CLI, updates the existing containers within the Container App. It will automatically deploy a new revision utilizing the new specs, and be online within a small period of time as the active revision. The Container App Environment itself does not need to be scaled up, as the default ConApp environment settings have it set to consumption based.

Reply:
Nice! Resolving this discussion and approving!